rpm_spec/manageiq.spec.in

```diff
@@ -59,23 +63,50 @@ EOF
 %install
 cd %{_builddir}
 
-### from core
+####################
+# Install manifest #
```

Comment:
I did a little rearranging here so that the manifest is installed first, since the other packages install into the manifest dir, so this ensures it's set up first and then the copy commands can be in their own sections.  I have a manifest for ansible-venv as well, hence why I did this.

If you want me break this refactoring out into a separate commit I can.